controllers/nodes/nodes_controller.go

```diff
@@ -387,13 +385,20 @@ func (r *ReconcileNodes) markForTermination(c *Cache, dynaKube *dynatracev1alpha
 		return nil
 	}
 
-	r.logger.Info("sending mark for termination event to dynatrace server", "ip", ipAddress, "node", nodeName)
-
 	if err = updateLastMarkedForTerminationTimestamp(c, &cachedNode, nodeName); err != nil {
 		return err
 	}
 
-	return r.sendMarkedForTermination(dynaKube, ipAddress, cachedNode.LastSeen)
+	if dk.FeatureDisableHostsRequests() {
```

Comment:
IIRC, the idea was to implement the feature flag logic for hosts API in dtclient. It would allow us to avoid this check throughout the code. Right now it is done twice...